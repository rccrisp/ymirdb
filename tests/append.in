set a 1 2 3
set b 4 5 6
append a b
list entries
forward a
backward b
sum a
len a
max a
append b 7
sum a
bye
