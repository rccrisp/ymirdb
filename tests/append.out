> ok

> ok

> ok

> b [4 5 6]
a [1 2 3 b]

> b

> a

> 21

> 6

> 6

> ok

> 28

> bye
